runtests.py

```diff
@@ -53,6 +53,7 @@
 
 import sys
 import os
+import builtins
```

Comment:
Is this needed?

Reply:
It's moved from below, but I have no idea if it's needed?
If all tests pass with it removed is that "good enough" to remove it?

Reply:
tests are happy without this so I removed it.